src/generic_ui/polymer/cloud-install.ts

```diff
@@ -130,5 +141,9 @@ Polymer({
   },
   ready: function() {
     this.ui = ui;
+    // ID of the latest attempt to create a server, used to distinguish
+    // between install failures that should be flagged to the user and
+    // failures owing to cancellation.
+    this.mostRecentCreateId = 0;
   }
 });
```

Comment:
Nice, 👍  on the comment, and thanks for explaining.

I wonder if something like `mostRecentCreateTime = (new Date()).getTime()` (and maybe even using `null` as the sentinel value) would be a little more self-explanatory, but totes no big d.